jbmc/unit/java_bytecode/java_bytecode_parser/ClassWithFields.java

```diff
@@ -0,0 +1,10 @@
+
+class ClassWithFields {
+  final boolean final1 = true;
+  final Boolean final2 = false;
```

Comment:
:bulb: I'd have probably picked different accessibility and/or different ways of setting them (i.e. in the constructor). You might also like to add tests for fields that hide inherited fields